projects/client/RabbitMQ.Client/src/client/impl/Command.cs

```diff
@@ -164,54 +134,27 @@ public void Transmit(int channelNumber, Connection connection)
             }
         }
 
-        public void TransmitAsSingleFrame(int channelNumber, Connection connection)
+        private void TransmitAsSingleFrame(int channelNumber, Connection connection)
```

Comment:
This seems to be a public API breaking change

Reply:
This method is never called by end user apps or libraries that depend on this client. See https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/350#issuecomment-329123265. So we can undo it but if someone's code depends on this, they are doing something really risky to begin with.